src/TokenParser/Render.php

```diff
@@ -1,19 +1,50 @@
 <?php
+/**
+ * @file
+ * Contains Drupal\twig_fractal\TokenParser\Render.
+ */
 
 namespace Drupal\twig_fractal\TokenParser;
 
 use Drupal\twig_fractal\Node;
 use Twig_Token;
 use Twig_TokenParser;
 
+/**
+ * Registers a 'render' tag and returns a new 'Render' node when the tag is parsed.
```

Comment:
Is the class registring it or implementing it? Is the class who returns the 'Render' or is a method. Maybe some more functional side description will be better for a class level doc?